.babelrc

```diff
@@ -1,9 +1,8 @@
 {
 	"presets": [
-		[ "latest", {
-			"es2015": {
-				"modules": false
-			}
-		} ]
+		[ "latest", "es2015" ]
```

Comment:
No, the last change is good 👍 

Yes, hard to know if it's working correctly. I guess we'll see when we import some function for `lodash`. We could check that it's not including the whole library